cli/config/cfg.go

```diff
@@ -0,0 +1,42 @@
+package config
```

Comment:
Why config structure is located in separate directiry('config') instead of ('configure') ?

Reply:
It is located in separate directory because modules.go needs it, and configure needs modules, so if config will be part of configure, configure and modules will be dependencies of each other.

Reply:
accepted